src/main/java/com/busify/busifyappproject/model/Tour.java

```diff
@@ -0,0 +1,81 @@
+package com.busify.busifyappproject.model;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+
+@Entity
+@Table(name = "Tour")
+public class Tour {
+
+    public Tour() {
+    }
+
+    public Tour(int id, Driver driver, LocalDate date, TourStatus status, Route route) {
+        this.id = id;
+        this.driver = driver;
+        this.date = date;
+        this.status = status;
+        this.route = route;
+    }
+
+    @Id
+    @GeneratedValue
+    @Column(name = "ID_Tour")
+    private int id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "ID_Driver")
+    private Driver driver;
+
+    @Column(name = "Date")
+    private LocalDate date;
```

Comment:
Change this to LocalDateTime. In  cases of tours and routes, I think it is better to have the time information as well.